app/code/community/Oyst/OneClick/etc/system.xml

```diff
@@ -572,6 +572,26 @@
                     </fields>
                 </reset>
 
+                <new_account translate="label">
+                    <label>Create account for new order</label>
```

Comment:
"Create a customer account for a new order"

French "Création d'un compte utilisateur lors d'une nouvelle commande"